src/it/scala/temple/generate/server/GoGeneratorIntegrationTestData.scala

```diff
@@ -11,26 +12,38 @@ object GoGeneratorIntegrationTestData {
     "user",
     "github.com/TempleEight/spec-golang/user",
     Seq.empty,
-    Set(CRUD.Create, CRUD.Read, CRUD.Update, CRUD.Delete),
+    ListMap(
+      CRUD.Create -> "INSERT INTO user_temple (id, name) VALUES ($1, $2) RETURNING id, name",
+      CRUD.Read   -> "SELECT id, name FROM user_temple WHERE id = $1",
+      CRUD.Update -> "UPDATE user_temple SET name = $1 WHERE id = $2 RETURNING id, name",
+      CRUD.Delete -> "DELETE FROM user_temple WHERE id = $1",
```

Comment:
I'd be tempted to make these strings come from the Postgres generator - especially given this is an integration test. Thoughts?

Reply:
Yeah fair good shout

Reply:
Unfortunately our Postgres gen doesn't support a couple things needed for this, so I have left them as strings ready to be updated when the necessary functionality is added to PostgresGenerator.